ObliviousHarmony/vscode-php-codesniffer
# Add `phpCodeSniffer.ignorePatterns` Option

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds a new `phpCodeSniffer.ignorePatterns` option that is used to decide whether or not a file should have diagnostics generated or it.

Fixes #19

### How to test the changes in this Pull Request:

1. Verify that `vendor` files are not processed by default
2. Modify `phpCodeSniffer.ignorePatterns` so `vendor` files will be processed
3. Verify that `vendor` files are now processed
